recipes/plateau/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "plateau" %}
+{% set version = "4.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/plateau-{{ version }}.tar.gz
+  sha256: fb7f06c99e63e83ab91af51482703a74e61ee5420242bebc5e9c859ebb19901c
+
+build:
+  skip: true  # [py2k]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools-scm
+  run:
+    - attrs
+    - click
+    - dask-core !=2021.5.1,!=2021.6.0
+    - decorator
+    - msgpack-python >=0.5.2
+    - numpy !=1.15.0,!=1.16.0
+    - pandas >=0.23.0,!=1.0.0
+    - prompt-toolkit
+    - pyarrow >=0.17.1,!=1.0.0,<5
+    - python
+    - pyyaml
+    - simplejson
+    - simplekv
+    - storefact
+    - toolz
+    - typing-extensions  # [py<38]
```

This is an empty package on py>=38 and omitting the selector here enables it being `noarch: python`.
```suggestion
    - typing-extensions
```